src/public/app/widgets/type_widgets/options/other/share_settings.ts

```diff
@@ -0,0 +1,51 @@
+import OptionsWidget from "../options_widget.js";
+import options from "../../../../services/options.js";
+import { t } from "../../../../services/i18n.js";
+import type { OptionMap, OptionNames } from "../../../../../../services/options_interface.js";
+
+const TPL = `
+<div class="card-body">
+    <h4>${t('Share Settings')}</h4>
+    
+    <div class="form-check">
+        <label class="form-check-label">
+            <input class="form-check-input" type="checkbox" name="redirectBareDomain" value="true">
+            ${t('Redirect bare domain to Share page')}
```

Comment:
the translations here will not work like this.
To make it work → you would need to add the strings as property to the `translation.json` file instead, e.g. like this:

```
"share": {
  "some_fitting_id": "Redirect bare domain to Share page" 
  …
}
```

you can take inspiration from e.g. the "Spellcheck" options section in that file.

Afterwards you would then use `t("share.some_fitting_id")` to get the message to display :-)